Asterisk/Manager.py

```diff
@@ -924,13 +924,14 @@ def PlayDTMF(self, channel, digit):
 
         return self._translate_response(self.read_response(id))
 
-    def QueueAdd(self, queue, interface, penalty=0):
-        'Add <interface> to <queue> with optional <penalty>.'
+    def QueueAdd(self, queue, interface, penalty=0, member_name=None):
+        'Add <interface> to <queue> with optional <penalty> and <member_name>.'
 
         id = self._write_action('QueueAdd', {
             'Queue': queue,
             'Interface': interface,
-            'Penalty': str(int(penalty))
+            'Penalty': str(int(penalty)),
+            'MemberName': str(member_name)
```

Comment:
You should not add the casting to string.

In case is not used the `member_name` parameter to call the QueueAdd this be translated a AMI Event like that

```
Event: QueueMemberAdded
Privilege: agent,all
SequenceNumber: 2
File: app_queue.c
Line: 6352
Func: add_to_queue
Queue: z1
Location: SIP/123123123
MemberName: None
StateInterface: 
Membership: dynamic
Penalty: 0
CallsTaken: 0
LastCall: 0
Status: 4
Paused: 0
```

And Asterisk will set as None the Member name in a Queue. 
**queue status**
```
z1 has 0 calls (max unlimited) in 'ringall' strategy (0s holdtime, 0s talktime), W:0, C:0, A:0, SL:0.0% within 0s
   Members: 
      None (SIP/123123123 from SIP/123123123) (ringinuse enabled) (dynamic) (Invalid) has taken no calls yet
```

You can use this  not as casting  or add conditional statement to  add this params to `write_action` only in case the value is not `None`